R-packages/covidcast/R/cor.R

```diff
@@ -2,8 +2,7 @@
 #'
 #' Computes correlations between two `covidcast_signal` data frames, allowing
 #' for slicing by geo location, or by time. (Only the latest issue from each
-#' data frame is used for correlations.) See the [correlations
-#' vignette](https://cmu-delphi.github.io/covidcast/covidcastR/articles/correlation-utils.html)
```

Comment:
why remove this?

Reply:
I meant to replace it with a link to \code{vignette("correlation-utils", package = "covidcast")} to link to the actual local vignette, but forgot to do that. Fixed in the updated, good catch!